l1-contracts/contracts/common/libraries/L2ContractHelper.sol

```diff
@@ -85,6 +85,35 @@ library L2ContractHelper {
         hashedBytecode = hashedBytecode | bytes32(bytecodeLenInWords << 224);
     }
 
+    /// @notice Validate the bytecode format and calculate its hash.
+    /// @param _bytecode The bytecode to hash.
+    /// @return hashedBytecode The 32-byte hash of the bytecode.
+    /// Note: The function reverts the execution if the bytecode has non expected format:
+    /// - Bytecode bytes length is not a multiple of 32
+    /// - Bytecode bytes length is not less than 2^21 bytes (2^16 words)
+    /// - Bytecode words length is not odd
+    function hashL2BytecodeCalldata(bytes calldata _bytecode) internal pure returns (bytes32 hashedBytecode) {
```

Comment:
In the L2Contracts its the other way around: 
https://github.com/matter-labs/era-contracts/blob/5b8b0785e4f7bae0b6d09d2dcd3d2b324204c16d/l2-contracts/contracts/L2ContractHelper.sol#L204

Reply:
okay, I'll make it more uniform

Reply:
It turned out changing l2-contracts folder was easier, so I will change it